package.json

```diff
@@ -1,16 +1,33 @@
 {
   "name": "restaurant",
   "version": "1.0.0",
+  "private": true,
   "description": "Restaurant Page",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "compile:sass": "node-sass sass/main.scss css/style.css -w",
+    "build": "webpack",
+    "start": "webpack serve --open"
   },
```

Comment:
Great work adding this dependency.  However, you should also add a `dev` dependency:

example: ``"dev": "webpack --mode development",``